src/pages/tours/create.tsx

```diff
@@ -0,0 +1,20 @@
+import AppPageLayout from "../../layouts/app-page-layout";
+import Typography from "@mui/material/Typography";
+import TourForm from "../../components/app/TourForm";
+import {Tour} from "../../types/tour";
+import {Point} from 'geojson'
+
+const NewTour = () => {
+    const tour: Tour = new Tour("", "", {} as Point, {} as Point, "", new Date(), new Date())
```

Comment:
This feels a bit ugly. I think we should use different DTOs for Updating or Creating, with one having default values or handle it in the form.

Reply:
Agreed. Specific DTOs added in another PR.